cookbooks/aws-parallelcluster-install/recipes/install.rb

```diff
@@ -20,11 +20,6 @@
 
 return if node['conditions']['ami_bootstrapped']
 
-# Calling user_ulimit will override every existing limit
```

Comment:
Please add in the commit message the fact that:
* you're renaming cluster_admin_user.rb without changes
* you're moving user_ulimit from install.rb into the new recipe

Reply:
sure will do